plutus-core/plutus-ir/src/PlutusIR/Analysis/Builtins.hs

```diff
@@ -162,3 +185,37 @@ defaultUniMatcherLike = Map.fromList
       vars <> TypeAppContext resTy resTyAnn (TermAppContext scrut scrutAnn branches)
     -- both branches have no args
     chooseListBranchArities = [[], []]
+
+-- See Note [Unserializable constants]
+defaultUniUnserializableConstants :: Some (ValueOf DefaultUni) -> Bool
+defaultUniUnserializableConstants = \case
+  Some (ValueOf DefaultUniBLS12_381_G1_Element _) -> True
+  Some (ValueOf DefaultUniBLS12_381_G2_Element _) -> True
+  Some (ValueOf DefaultUniBLS12_381_MlResult _) -> True
+  _ -> False
```

Comment:
You could tediously spell out all the other cases here so that if we add more types to the universe we'd be forced to look at this.

Reply:
I could, but I'm not sure it's worth it. This is a weird special case...